src/System.ComponentModel.Composition/tests/Microsoft/Internal/AssumesTests.cs

```diff
@@ -0,0 +1,237 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace Microsoft.Internal
+{
+    [ActiveIssue(123456789)]
```

Comment:
I guess this is a temporary issue number?

Reply:
yes